.github/actions/build-cache/action.yml

```diff
@@ -56,3 +56,12 @@ runs:
         key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
         restore-keys: |
           ${{ runner.os }}-pnpm-
+    - name: Restore Playwright Browser Cache
```

Comment:
Can we move this to the e2e test action itself, so we're not restoring this for tests that don't use it?

Reply:
ah yeah! good call, will do